aii-core/src/main/perl/Shellfe.pm

```diff
@@ -30,8 +30,9 @@ use File::Path qw(mkpath rmtree);
 use File::Basename qw(basename);
 use DB_File;
 use Readonly;
-
+use Parallel::ForkManager 0.7.6;
```

Comment:
out of curiosity, what is so special about `0.7.6`. el6 has `0.7.9` in epel.
@ned21 @jrha @jouvin do we need el5 support for aii?